pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,228 @@
-                            zebra: ["even", "odd"],
-                            filter_reset: "#resetFilter",
-                            filter_cssFilter: "form-control",
-                            filter_functions: {
-                                3: {
-                                    <% all_tags.forEach(function(tag) { %>
-                                    "<%= escapeRegExp(tag.name) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(tag.name) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% if (locals.course_instance) { %>
-                                5: {
-                                    <% all_assessments.forEach(function(assessment) { %>
-                                    "<%= escapeRegExp(assessment.label) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(assessment.label) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% } %>
-                            },
-                        },
-                    });
-                });
-            </script>
+          </div>
+        </div>
+
+        <table id="questionsTable"
+               data-data="<%= JSON.stringify(questions) %>"
+               data-classes="table table-sm table-hover table-bordered"
+               data-thead-classes="thead-light"
+               data-filter-control="true"
+               data-show-filter-control-switch="true"
+               data-show-search-clear-button="true"
+               data-show-columns="true"
+               data-show-columns-toggle-all="true"
+               data-show-button-text="true"
+               data-pagination="true"
+               data-pagination-v-align="top"
+               data-page-list="[50,100,200,500,unlimited]"
+               data-page-size="50"
+               data-smart-display="false"
+               data-show-extended-pagination="true"
+               data-toolbar=".fixed-table-pagination"
+               data-sticky-header="true">
+          <thead>
+            <tr>
+              <th data-field="qid"
+                  data-sortable="true"
+                  data-class="align-middle sticky-column"
+                  data-formatter="qidFormatter"
+                  data-filter-control="input"
+                  data-switchable="true">QID</th>
+              <th data-field="title"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-filter-control="input"
+                  data-filter-custom-search="genericFilterSearch"
+                  data-switchable="true">Title</th>
+              <th data-field="topic"
+                  data-sortable="true"
```

Comment:
Sorting by topic seems incorrect.

<img width="126" alt="image" src="https://user-images.githubusercontent.com/875833/128928201-933101f4-f333-495d-993f-d463cb6f84e1.png">

Reply:
Also, I need to click the topic sort button 3 times to cycle between the different states. It seems to do (1) sort ascending, (2) nothing, (3) sort descending.